datadog_lambda/trigger.py

```diff
@@ -0,0 +1,182 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+
+import base64
+import gzip
+import json
+from io import BytesIO, BufferedReader
+
+
+EVENT_SOURCES = {
+    "aws:dynamodb": "dynamodb",
+    "aws:kinesis": "kinesis",
+    "aws:s3": "s3",
+    "aws:sns": "sns",
+    "aws:sqs": "sqs",
+}
+
+
+def get_first_record(event):
+    records = event.get("Records")
+    if records and len(records) > 0:
+        return records[0]
+
+
+def read_event_source(event):
```

Comment:
Can you make the function name more explicit/specific? I don't see a good difference between `read_event_source` and `get_event_source`, and `get_event_source` calling `read_event_source` doesn't really tell a good story on what is the relationship here.

Reply:
Actually, do you even need this function at all? It might be better to just put these code back into get_event_source?